File: Heat/Migrations/201509261312561_rename_filenameMedium.vb
Imports System
Imports System.Data.Entity.Migrations
Imports Microsoft.VisualBasic

Namespace Migrations
    Public Partial Class rename_filenameMedium
        Inherits DbMigration
    
        Public Overrides Sub Up()
            AddColumn("dbo.Media", "OriginalFilename", Function(c) c.String())
            DropColumn("dbo.Media", "FileName")
        End Sub
        
        Public Overrides Sub Down()
            AddColumn("dbo.Media", "FileName", Function(c) c.String())
            DropColumn("dbo.Media", "OriginalFilename")
        End Sub
    End Class
End Namespace

File: Heat.ConvertedToC#/Migrations/201509261312561_rename_filenameMedium.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Xml.Linq;
using System.Diagnostics;
using System.Collections.Specialized;
using System.Configuration;
using System.Text;
using System.Text.RegularExpressions;
using System.Web;
using System.Web.Caching;
using System.Web.Mvc;
using System.Web.Mvc.Ajax;
using System.Web.Mvc.Html;
using System.Web.Routing;
using System.Web.SessionState;
using System.Web.Security;
using System.Web.Profile;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;
using System.Data.Entity.Migrations;

namespace Heat.Migrations
{
	public partial class rename_filenameMedium : DbMigration
	{

		public override void Up()
		{
			AddColumn("dbo.Media", "OriginalFilename", c => c.String());
			DropColumn("dbo.Media", "FileName");
		}

		public override void Down()
		{
			AddColumn("dbo.Media", "FileName", c => c.String());
			DropColumn("dbo.Media", "OriginalFilename");
		}
	}
}
